src/components/calcite-input/calcite-input.tsx

```diff
@@ -516,17 +521,20 @@ export class CalciteInput implements LabelableComponent, FormComponent {
 
     const inputMax = this.maxString ? parseFloat(this.maxString) : null;
     const inputMin = this.minString ? parseFloat(this.minString) : null;
-    const valueNudgeDelayInMs = 500;
+    const valueNudgeDelayInMs = 100;
 
     this.incrementOrDecrementNumberValue(direction, inputMax, inputMin, nativeEvent);
 
-    this.nudgeNumberValueIntervalId = setInterval(() => {
-      this.incrementOrDecrementNumberValue(direction, inputMax, inputMin, nativeEvent);
-    }, valueNudgeDelayInMs);
+    this.nudgeNumberValueTimeoutId = setTimeout(() => {
+      this.nudgeNumberValueIntervalId = setInterval(() => {
+        this.incrementOrDecrementNumberValue(direction, inputMax, inputMin, nativeEvent);
```

Comment:
Instead of the wrapping timeout, would it be simpler to use a local var as a flag to ignore the first interval call?